examples/deployment_ec2_instance/README.md

```diff
@@ -0,0 +1,11 @@
+# Deployment with an EC2 Instance example
+
+This example code shows how easy it is to build an application's infrastrucute using ec2 instances and an Elastic Cloud deployment communicating securly using traffic filters
```

Comment:
From a conversation with cloud writers I remember something was said about avoiding using terms like "easy" or "simple". Could you confirm @alaudazzi @nrichers ?

Reply:
@karencfv These two terms are not contemplated in the new [Writing style guide](https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide). I would just reword the sentence like this:

> This example code shows how to build an application's infrastructure using ec2 instances ....